### ISSUE TITLE
No way to "update if newer" on installed files / Rsync not respecting Timestamps from internal scripts

### ISSUE THREAD

**Issue by bobpies** [packagesdev/packages#101]
This problem is twofold

Currently, when you select and install your files it overwrites everything - there is no way to tell the installer only to update the files if they are newer than what is already installed.

To circumvent this problem i installed the files to a new location and used "RSYNC -rvc --update" on the files to copy them to the new location IF they are newer. 
This is where we hit the second problem - This command works perfectly fine in terminal - copies only the new files.
However, including it as a postinstall script doesn't work at all - it copies all of the files no matter what. So confusing.

I've added sudo to the command and i've added -t aswell to copy timestamps but no luck - just copies and overwrites everything.

So this is either a feature request to add a "install if newer" flag
or it's a bug fix request for running Rsync as a script inside the PKG as it ignores the timestamps on files.

**Comment by packagesdev** [comment]
The .pkg installation format only supports versioning based on the bundle and framework versions.

Are you trying to install some files only if there are not already on disk? Or is it really a newer criteria? If it's the first case, you may find this Q&A useful:

http://s.sudre.free.fr/Software/Packages/Q&A_9.html

**Comment by bobpies** [comment]
Yes I’m trying to a) install a folder of files and b) update them, if they exist. 
but I don’t want them overwritten in every install.

I use NSIS on windows and this has an update if newer option I can just turn on at this stage. 

Running the rsync as a script doesn’t seem to work as expected. 

For now im having a bit of luck with moving all of the existing files to another location - installing the installer files overwriting everything, and then using rsync to copy if newer on top from the backups - this seems to have better luck with picking up the time stamps for some reason ..

**Comment by donmontalvo** [comment]
@rudyrichter not trying to be Debbie Downer, but curious why not just replace and avoid risk>
